blazor/rich-text-editor/tools/dialogSettings.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Customization of Blazor Rich Text Editor Dialog | Syncfusion
+description: Check out and learn here all about DialogSettings in Syncfusion Blazor RichTextEditor component and more.
+platform: Blazor
+control: RichTextEditor
+documentation: ug
+---
+
+# Customization of Blazor Rich Text Editor Dialog
+
+The integration of the child component allows for improved customization of the RichTextEditor in the Blazor component. With this, you can now directly control the RichTextEditor's dialogs.
+
+{% tabs %}
+{% highlight cshtml %}
+
+{% include_relative code-snippet/dialog-settings.razor %}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Blazor RichTextEditor DialogSettings](../images/dialog-settings.png)
+
+## The child component exposes three essential properties:
```

Comment:
Dont need this, You can add the tag in above section

Reply:
Addressed